### ISSUE TITLE
Fix deprecation of joinedload_all() in sqa

### ISSUE THREAD

**Issue by chrzyki** [clld/clld#184]
Apparently, this has been deprecated for quite some time (since 0.9?), so a removal in the near future is not unlikely. Chained calls like

```python
joinedload_all(
    common.Contribution.valuesets,
    common.ValueSet.values,
    common.Value.domainelement),
joinedload_all(
    common.Contribution.references,
    common.ContributionReference.source),
joinedload(common.Contribution.data),
)
```
would have to be rewritten using multiple chained `joinedload().joinedload(). ...` etc.

**Comment by xrotwang** [comment]
Could you check how often this appear in `clld` - maybe we could roll this fix into 4.5?

**Comment by chrzyki** [comment]
Will do!
